content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -2,24 +2,63 @@
+leads and opportunities.
 
+Enable multi teams
+==================
+
+To allow salespeople to be assigned to more than one sales team, the :guilabel:`Multi Teams` setting
+needs to be enabled. First, navigate to :menuselection:`CRM --> Configuration --> Settings`. Under
+the :guilabel:`CRM` section, tick the checkbox labeled :guilabel:`Multi Teams`. Then, click
+:guilabel:`Save` at the top-left of the page.
+
+.. image:: multi_sales_team/multi-team-setting.png
+   :align: center
+   :alt: The Multi Teams setting in the CRM app.
+
+.. _crm/assignment-rules:
+
+Enable assignment rules
+=======================
+
+To periodically :doc:`assign leads <../track_leads/lead_scoring>` based on rules, the
+:guilabel:`Rule-Based Assignment` feature needs to be enabled. Navigate to :menuselection:`CRM -->
+Configuration --> Settings`. Under the :guilabel:`CRM` section, tick the checkbox labeled
+:guilabel:`Rule-Based Assignment`.
+
+This causes a new field, :guilabel:`Running` to appear. Select an option from the drop-down to
+manage how assignments are triggered.
+
+The rule-based assignment feature can be set to run :guilabel:`Manually`, meaning an Odoo user must
+manually trigger the assignment, or :guilabel:`Repeatedly`, meaning Odoo automatically triggers the
+assignments.
+
+Selecting :guilabel:`Repeatedly` causes additional fields to appear. Use the :guilabel:`Repeat
+every` fields to select how frequently the rules should be triggered. The :guilabel:`Next Run` field
+automatically updates, thought it can be manually updated.
+
+.. image:: multi_sales_team/rule-based-assignment.png
+   :align: center
+   :alt: The Rule-Based Assignment setting enabled in the CRM app.
+
+.. seealso::
+   For more information on assigning leads, see :doc:`assign leads <../track_leads/lead_scoring>`.
 
 Create a new sales team
 =======================
 
 To create a new sales team, go to :menuselection:`CRM --> Configuration --> Sales Teams`, then
-click :guilabel:`Create`.
+click :guilabel:`New`.
 
-On the creation page, set an :guilabel:`Email Alias` to automatically generate a lead/opportunity
-for this sales team every time a message is sent to that unique email address. Choose whether to
-accept emails from :guilabel:`Everyone`, :guilabel:`Authenticated Partners`, or :guilabel:`Followers
-Only`.
+On the blank record enter a name in the :guilabel:`Sales Team` field.
 
-Set an :guilabel:`Invoicing Target` if this team has specific monthly revenue goals. Set a
-:guilabel:`Domain` to assign leads/opportunities to this sales team based on specific filters, such
-as country, language, or campaign.
+Next, select a :guilabel:`Team Leader` from the drop-down list.
+
+Set an :guilabel:`Email Alias` to automatically generate a lead/opportunity for this sales team when
+a message is sent to that unique email address. Choose whether to accept emails from
+:guilabel:`Everyone`, :guilabel:`Authenticated Partners`, or :guilabel:`Followers Only`.
```

Comment:
I'm also seeing an option for 'Authenticated Employees'...might be worth a quick double-check on that during your revision